config/ConfigExample/RemoteConfigViewController.swift

```diff
@@ -48,41 +49,41 @@ class RemoteConfigViewController: UIViewController {
 
   // MARK: - Firebase 🔥
 
+  // Add additional keys that are in the console to see them in the log after fetching the config.
+  struct QSConfig: Codable {
+    let topLabelKey: String
+    let bottomLabelKey: String
+    let freeCount: Int
+  }
+
   /// Initializes defaults from `RemoteConfigDefaults.plist` and sets config's settings to developer mode
   private func setupRemoteConfig() {
     remoteConfig = RemoteConfig.remoteConfig()
-    remoteConfig.setDefaults(fromPlist: "RemoteConfigDefaults")
+    // This is an alternative to remoteConfig.setDefaults(fromPlist: "RemoteConfigDefaults"))
+    try? remoteConfig.setDefaults(from: QSConfig(topLabelKey: "myTopLabel",
+                                                 bottomLabelKey: "Buy one get one free!",
+                                                 freeCount: 4))
```

Comment:
nit: handle the returned error instead of ignoring it

Reply:
Done.